content/protocol-adapters/index.textile

```diff
@@ -0,0 +1,44 @@
+---
+title: Protocol Adapters
+meta_description: "Clients can use the Ably network using SSE and MQTT protocol adapters. This is especially useful where an Ably Client Library SDK is not available for your language of choice, or where platform resource constraints prohibit use of a Client Library SDK."
+meta_keywords: "SSE, MQTT, IoT, thin client, protocol adapter, lightweight clients, Arduino, Raspberry Pi, wearable"
+section: protocol-adapters
+index: 0
+jump_to:
+  Help with:
+    - MQTT#mqtt
+    - SSE#sse
+---
+
+While the recommended method for connecting to Ably is through the use of the Ably Client Library SDKs, there are certain situations where this is not possible, due to lack of platform support, or platform resource constraints. In this case, certain clients may be supported through the use of Ably protocol adapters. 
```

Comment:
It seems a bit repetitive to have this para and the next explaining almost the same thing, wdyt?

Reply:
I've re-written the introductory section to account for this.